Task150/Task150/Task150/Pages/MyWishlistPage.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+using Task150.HelperPage;
+
+namespace Task150.Pages
+{
+    internal class MyWishlistPage : Page
+    {
+        private string NAME_WISHLIST = "MyWishlist";
+
+        [FindsBy(How = How.CssSelector, Using = "tbody > tr .wishlist_delete > .icon")]
+        private IList<IWebElement> WishlistListsDeleteButton;
+
+        [FindsBy(How = How.Id, Using = "name")]
+        private IWebElement NameWishlistField;
+
+        [FindsBy(How =How.Id, Using = "submitWishlist")]
+        private IWebElement SaveWishlistButton;
+
+        public MyWishlistPage(IWebDriver driver) : base(driver)
+        {
+            PageFactory.InitElements(driver, this);
+        }
+
+        public bool IsThisPage()
+        {
+            var byPageHeadingTitle = By.ClassName("page-heading");
+
+            return _driver.WaiterByElementIsDisplay(byPageHeadingTitle)
+                   && _driver.FindElement(byPageHeadingTitle)
+                       .Text
+                       .ToLower()
+                       .Equals("my wishlists");
+        }
```

Comment:
The same about page opened validation. Try to verify based on Title of page or something like this. If you need to verify that some element is displayed then just call corresponding Waiter. But if you need to verify that necessary page was opened then check Title of Url of this page.

Reply:
> The same about page opened validation. Try to verify based on Title of page or something like this. If you need to verify that some element is displayed then just call corresponding Waiter. But if you need to verify that necessary page was opened then check Title of Url of this page.

And what should I do if there is no Title on the page